web/profiles/joinup/joinup.info.yml

```diff
@@ -17,6 +17,7 @@ dependencies:
   - file
   - help
   - image
+  - joinup_document
```

Comment:
Yes but aren't these content types able to attach to other entities (e.g. solutions)?